js/src/util/ethlink.js

```diff
@@ -14,18 +14,28 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
-import { decode as decodeBase58, encode as encodeBase58 } from 'bs58';
-
+// Using base-x since we started with base-58 originally. This one allows the
+// very slight advantage of using custom dictionaries (useful in the case of
+// base-32 where there are multiples available)
+import base32 from 'base32.js';
+
+// base32 alphabet should match the Rust implementation
+// https://github.com/andreasots/base32/blob/master/src/base32.rs
```

Comment:
We are not even close to having that - we are still actually in development here and finding what works between the platforms without decoding issues.